core/schema/container.go

```diff
@@ -484,13 +470,13 @@ func (s *containerSchema) Install() {
 
 		dagql.Func("stdout", s.stdout).
 			View(AllVersion).
-			Doc(`The output stream of the last executed command.`,
-				`Returns an error if no command was set.`),
+			Doc(`The buffered standard output stream of the last executed command`,
+				`Returns an error if no command was executed`),
 
 		dagql.Func("stderr", s.stderr).
 			View(AllVersion).
-			Doc(`The error stream of the last executed command.`,
-				`Returns an error if no command was set.`),
+			Doc(`The buffered standard error stream of the last executed command`,
```

Comment:
mini nit: this isn't really a stream, it's the entire output buffer.

Reply:
Yeah that's why it says "buffered".